### PR TITLE
Added in font validation & allowed font types

### PR DESCRIPTION
**Pull request by jane-dylla**
Added in font validation
Added in list of allowed fonts
Added font files for Montserrat font
Added code to register font files

### CODE REVIEW

File: HTMLToQPDF/Components/HTMLComponent.cs [hunk 1]
```diff
@@ -52,6 +53,14 @@ internal class HTMLComponent : IComponent
 
         public void Compose(IContainer container)
         {
+            var fontPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Assets\\Fonts");
```
Comment (ADNewsom09) [inline HTMLToQPDF/Components/HTMLComponent.cs]: you want pathseperatorcharacter instead of \\?

File: HTMLToQPDF/Utils/FontFamilyUtils.cs [hunk 2]
```diff
@@ -5,6 +5,20 @@ namespace HTMLToQPDF.Utils
 {
     internal static class FontFamilyUtils
     {
+        private static HashSet<string> availableFonts = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```
Comment (ADNewsom09) [inline HTMLToQPDF/Utils/FontFamilyUtils.cs]: could be readonly?

Comment (jane-dylla) [reply HTMLToQPDF/Utils/FontFamilyUtils.cs]: updated in most recent commit

File: HTMLToQPDF/Components/HTMLComponent.cs (hunk 1 above)
Comment (jane-dylla) [reply HTMLToQPDF/Components/HTMLComponent.cs]: updated in most recent commit

**Review by ADNewsom09 — APPROVED** [review]

**Pull request merged**